src/vmware/azext_vmware/azext_metadata.json

```diff
@@ -1,4 +1,4 @@
 {
     "azext.isPreview": false,
-    "azext.minCliCoreVersion": "2.54.0"
+    "azext.minCliCoreVersion": "2.67.0"

```

Comment:
Bumping the minimum AZ CLI version from 2.54 to 2.67 probably warrents bumping the miror version to `7.2.0`.